websites/D/Disney+/presence.ts

```diff
@@ -57,22 +57,17 @@ presence.on("UpdateData", async () => {
 						presenceData.state =
 							document.querySelector(".subtitle-field")?.textContent;
 
-						const paused = !!document.querySelector("[aria-label='Play']");
+						const paused = !!document.querySelector("[aria-label='Play']"),
+							timeRemaining = document.querySelector(
+								".time-remaining-label"
+							).textContent;
```

Comment:
So it doesn't error when it doesn't exist?
```suggestion
							)?.textContent;
```

Reply:
It will and that's better